clients/client/src/main/java/org/eclipse/sw360/clients/config/CommonUtils.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright Siemens AG, 2021.
+ * With modifications by Bosch Software Innovations GmbH, 2016
+ * Part of the SW360 Portal Project.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.eclipse.sw360.clients.config;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Optional;
+import java.util.Properties;
+
+import org.apache.commons.io.IOUtils;
+
+public class CommonUtils {
+
+    public static final String SYSTEM_CONFIGURATION_PATH = "/etc/sw360";
+
+    public static Properties loadProperties(Class<?> clazz, String propertiesFilePath) {
+        return loadProperties(clazz, propertiesFilePath, true);
+    }
+
+    public static Properties loadProperties(Class<?> clazz, String propertiesFilePath, boolean useSystemConfig) {
+        Properties props = new Properties();
+
+        try (InputStream resourceAsStream = clazz.getResourceAsStream(propertiesFilePath)) {
+            if (resourceAsStream == null)
+                throw new IOException("cannot open " + propertiesFilePath);
+
+            props.load(resourceAsStream);
+        } catch (IOException e) {
+        }
```

Comment:
May be we can print corresponding exception messages, instead of keeping the catch block empty. 
If agree, then please check the below catch blocks as well.